recipes/deepchem/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "deepchem" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepchem/deepchem/archive/{{ version }}.tar.gz
+  sha256: eec0d4dfa23e234d26d171ef20c62a699052a1bc230bbace341047e0c2673030
+
+build:
+  number: 0
+  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  skip: True   # [win or py==27]
```

Comment:
Can this be noarch Python?

Reply:
In principle yes but the windows build fails with `Unsatisfiable dependencies for platform win-64: {'xgboost'}.` 

See: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=119464&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=78a0099b-3f13-5ae9-c1f0-24f71d1eac20

Reply:
noarch only needs to build on linux, so you shouldn't worry about the windows failures

Reply:
@scopatz, have updated and the linux build is passing.